mtp_noms_ops/apps/security/views/check.py

```diff
@@ -316,6 +320,13 @@ def form_valid(self, form):
                 ui_message = _('Credit rejected')
             messages.info(self.request, ui_message)
 
-            return HttpResponseRedirect(self.list_url)
+            redirect_url = self.request.POST.get('redirect_url', self.list_url)
```

Comment:
i think it'd be better to use the validated/pre-populated form value here?
```python
redirect_url = form.cleaned_data['redirect_url']
```

Reply:
🤔 is it the same? It should. I'll give it a try, thanks for the suggestion ☺️

Reply:
https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/608/commits/8e791d7a227f5ed51ed8b58b8719984f78aa64d7

Reply:
It seems to work fine 👍